app/views/recipes/_recipe.html.erb

```diff
@@ -3,6 +3,35 @@
     <%= recipe.name %>
   </h1>
 
+  <p class="my-5">
+    <strong class="block font-medium mb-1">Yield:</strong>
+    <%= simple_format(recipe.yield) %>
+  </p>
+
+  <p class="my-5">
+    <strong class="block font-medium mb-1">Prep Time:</strong>
+    <%= simple_format(recipe.prep_time) %>
+  </p>
+
+  <p class="my-5">
+    <strong class="block font-medium mb-1">Cook Time:</strong>
+    <%= simple_format(recipe.cook_time) %>
+  </p>
+
+  <p class="my-5">
+    <strong class="block font-medium mb-1">Rating:</strong>
+    <%= recipe.rating %>
+  </p>
+
+  <p class="my-5">
+    <strong class="block font-medium mb-1"> <%= recipe.is_favorite == true ?   "Favorite: yes"  : " " %></strong>
+  </p>
+
+  <p class="my-5">
+    <strong class="block font-medium mb-1">Description:</strong>
+    <%= simple_format(recipe.description) %>
+  </p>
```

Comment:
Do you want that logic on every field or just like notes and description? I already have something like that on is_favorite.

Reply:
I think it would be useful on all fields except name which is required and maybe ingredients/directions which aren't required but a recipe is incomplete without

Reply:
One way to accomplish this is to wrap the section in an if-block:

```
<% if recipe.description.present? %>
  <p class="my-5">
    <strong class="block font-medium mb-1">Description:</strong>
    <%= simple_format(recipe.description) %>
  </p>
<% end %>
```